plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/preferences/PrefPageToolbarCustomization.java

```diff
@@ -187,7 +188,7 @@ public PrefPageToolbarCustomization() {
         toolBarImage = findBundleImage(PlatformUI.PLUGIN_ID, "$nl$/icons/full/obj16/toolbar.png"); //$NON-NLS-1$
         
         toolBarNodes = ToolBarConfigurationRegistry.getInstance().getKnownToolBars().stream()
-            .map(ToolBarNode::new).collect(Collectors.toList());
+            .sorted(Comparator.comparing(ToolBarConfigurationDescriptor::getName)).map(ToolBarNode::new).collect(Collectors.toList());
```

Comment:
Please split it over several lines to make it more readable

```suggestion
            .sorted(Comparator.comparing(ToolBarConfigurationDescriptor::getName))
            .map(ToolBarNode::new)
            .collect(Collectors.toList());
```